dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.internal.jca-1.6.feature

```diff
@@ -22,3 +22,4 @@ IBM-API-Package: \
   com.ibm.ws.jca.feature
 kind=ga
 edition=base
+WLP-Activation-Type: parallel
```

Comment:
Do you also want to update com.ibm.websphere.appserver.javax.connector.internal-1.6.feature?